src/aoc2020/day1.clj

```diff
@@ -0,0 +1,143 @@
+(ns aoc2020.day1
+  (:require [aoc2020.util :refer [file->seq]]))
```

Comment:
Please refer to the clojure style guide.

ALL of the function are still not following team conventions.

```
(defn <name>
   [docstring]
   <param vector>

```

Reply:
This namespace is STILL not properly indented. 
* Use auto-indent hotkeys built into IntelliJ.
* **Verify you have IntelliJ settings matching the onboarding guide**

Reply:
As mentioned several times already, comments should not exist outside of the REPL block. These tend to not get updated, and the allow for sloppier code.

**Do not add comments in line with functions (i.e. `;; some comment`).**

Reply:
> As mentioned several times already, comments should not exist outside of the REPL block. These tend to not get updated, and the allow for sloppier code.

done

Reply:
> This namespace is STILL not properly indented.
> 
> * Use auto-indent hotkeys built into IntelliJ.
> * **Verify you have IntelliJ settings matching the onboarding guide**

May I ask where is the improper indent? (I think I have followed the onboarding guide and used auto-indent)

Reply:
> Please refer to the clojure style guide.
> 
> ALL of the function are still not following team conventions.
> 
> ```
> (defn <name>
>    [docstring]
>    <param vector>
> ```

done.

Reply:
> > This namespace is STILL not properly indented.
> > 
> > * Use auto-indent hotkeys built into IntelliJ.
> > * **Verify you have IntelliJ settings matching the onboarding guide**
> 
> May I ask where is the improper indent? (I think I have followed the onboarding guide and used auto-indent)

Did you go through the onboarding setup guide and verify this?

Reply:
This is still broken.

Look up the auto-indent hotkey in IntelliJ and make sure you are using it.